pkg/kube/util/names/names.go

```diff
@@ -82,6 +74,30 @@ func CalculateEJobOutputSecretPrefixAndName(secretType DeploymentSecretType, dep
 	return prefix + ".", finalName
 }
 
+var allowedKubeChars = regexp.MustCompile("[^-a-z0-9]*")
+
+// ContainerName k8s valid container name: [a-z0-9]([-a-z0-9]*[a-z0-9])?
+func ContainerName(name string) string {
```

Comment:
How about calling it `Sanitize`?

Reply:
It doesn't seem to be in the core. Do you mean this one for HTML https://github.com/kennygrant/sanitize? 
I wouldn't bring another dependency in when a simple regex is enough.

Reply:
Oh no, just the naming. `Sanitize` sounds more correct than `ContainerName`.

Reply:
Hm, yes I can rename it. Sanitize sounds great.

 I was just wondering if it's too early to find a general name, the func is only used for the container names now. As far as I understand the other resources would also benefit from this, but are implemented slightly different now, i.e. services have a dash in between name and md5. So I expected some clean up in the future...

Reply:
Yes, we can reuse it for various k8s resource naming. Thanks for renaming it.